jobs/ci-kubernetes-e2e-gce-gci-latest-upgrade-etcd.env

```diff
@@ -0,0 +1,15 @@
+### job-env
+
+STORAGE_BACKEND=etcd2
+ETCD_VERSION=2.2.1
+ETCD_IMAGE=2.2.1
+
+E2E_OPT=--check_version_skew=false
+E2E_UPGRADE_TEST=true
+GINKGO_UPGRADE_TEST_ARGS=--ginkgo.focus=\[Feature:EtcdUpgrade\] --etcd-upgrade-storage=etcd3 --etcd-upgrade-version=3.0.17
+JENKINS_PUBLISHED_SKEW_VERSION=ci/latest
```

Comment:
Since we most probably be upgrading 1.5 clusters, I think we should use 1.5 branch for the testing purpose.